requirements.txt

```diff
@@ -48,7 +48,7 @@ qtconsole==4.3.1
 scipy==0.19.1
 simplegeneric==0.8.1
 six==1.11.0
-tensorflow==1.3.0
+tensorflow-gpu==1.3.0
```

Comment:
Can non-GPU users still use `tensorflow` instead?

Reply:
Yeah, sorry I didn't mean to commit that part